app/src/main/java/com/sun/colornotetaking/ui/home/HomePresenter.java

```diff
@@ -0,0 +1,81 @@
+package com.sun.colornotetaking.ui.home;
+
+import com.sun.colornotetaking.data.OnDataLoadingCallback;
+import com.sun.colornotetaking.data.TaskRepository;
+import com.sun.colornotetaking.data.model.Task;
+
+import java.util.List;
+
+public class HomePresenter implements HomeContract.Presenter {
+
+    private TaskRepository mTaskRepository;
+    private HomeContract.View mView;
+
+    public HomePresenter(TaskRepository taskRepository, HomeContract.View view) {
+        mTaskRepository = taskRepository;
+        mView = view;
+    }
+
+    @Override
+    public void getPinedTasks() {
+        mTaskRepository.getTasks(true, new OnDataLoadingCallback<List<Task>>() {
+            @Override
+            public void onDataLoaded(List<Task> data) {
+                if (data.size() == 0) mView.showEmptyTasks(null);
```

Comment:
showEmptyTask thì tát nhiên null rồi
```suggestion
                if (data.size() == 0) mView.showEmptyTasks();
```
thiếu khối lệnh